soh/soh/Enhancements/mods.cpp

```diff
@@ -11,12 +11,6 @@ extern "C" {
 #include "functions.h"
 extern SaveContext gSaveContext;
 extern PlayState* gPlayState;
-extern void Play_PerformSave(PlayState* play);
-extern s32 Health_ChangeBy(PlayState* play, s16 healthChange);
-extern void Rupees_ChangeBy(s16 rupeeChange);
-extern Actor* Actor_Spawn(ActorContext* actorCtx, PlayState* play, s16 actorId, f32 posX, f32 posY, f32 posZ,
-                            s16 rotX, s16 rotY, s16 rotZ, s16 params, s16 canRandomize);
-extern void Inventory_ChangeEquipment(s16 equipment, u16 value);
```

Comment:
These don't need to be here because functions.h is already included. I've removed these earlier but I guess they were put back during merges, so I'm removing them here again.